pallets/did/src/lib.rs

```diff
@@ -165,27 +165,55 @@ pub mod pallet {
 			+ Dispatchable<Origin = <Self as Config>::Origin, PostInfo = PostDispatchInfo>
 			+ GetDispatchInfo
 			+ DeriveDidCallAuthorizationVerificationKeyRelationship;
+
 		/// Type for a DID subject identifier.
 		type DidIdentifier: Parameter + Default + DidVerifiableIdentifier;
+
 		#[cfg(not(feature = "runtime-benchmarks"))]
 		/// Origin type expected by the proxied dispatchable calls.
 		type Origin: From<DidRawOrigin<DidIdentifierOf<Self>>>;
+
 		#[cfg(feature = "runtime-benchmarks")]
 		type Origin: From<RawOrigin<DidIdentifierOf<Self>>>;
 		type EnsureOrigin: EnsureOrigin<Success = DidIdentifierOf<Self>, <Self as frame_system::Config>::Origin>;
 		/// Overarching event type.
 		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
+
+		/// Maximum number of total public keys which can be stored per DID key
+		/// identifier. This includes the ones currently used for
+		/// authentication, key agreement, attestation, and delegation. It also
+		/// contains old attestation keys which cannot issue new attestations
+		/// but can still be used to verify previously issued attestations.
+		#[pallet::constant]
+		type MaxPublicKeysPerDidKeyIdentifier: Get<u32>;
```

Comment:
I think we want to remove `KeyIdentifier` from the name, right? We just reach the maximum number of public keys for a given DID.
```suggestion
		type MaxPublicKeysPerDid: Get<u32>;
```

Reply:
b90ed76f0a15f2cc9a30bbdb26f0593222eafbb0